### DISCUSSION TITLE
Change card background color based on average status?

### DISCUSSION THREAD

**Discussion by DSew20** [dylandoamaral/uptime-card#157] [Q&A]
Hello - is there a way to set the card color based on average status?

I would like to create a small "summary" card that shows the uptime status for the last hour.  So, if up 100% the card is green.  If up xx% of time orange, otherwise red?  I will then add a grid of cards to show the overall status of my network.

Happy to use the default % thresholds, but want to color the card not just the text.  Any ideas, tried card-mod but struggling to see how I get the current uptime status from the card itself?

**Comment by DSew20** [comment]
Ive got this far.  But, how do I get the average value out of the card?  I've tried values.average but doesn't seem to work.

Any pointers?

```
card_mod:
  style: |
    ha-card {
      {%- set cardstat = variables.average -%}
      {%- if cardstat >= 99 -%}
        background-color: #45C669
      {%- elif cardstat >= 80 -%}
        background-color: #C6B145
      {%- elif cardstat < 80 -%}
        background-color: #C66445
      {%- else %}
        background-color: #C9C9C9
      {%- endif %}
    }
```

**Reply by dylandoamaral** [reply]
I don't really know. I won't add this feature officialy but you may have better chance updating uptime-card.js directly. The only issue with this method is that the changes will be erased if I update the card :/.
